src/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs

```diff
@@ -183,7 +183,6 @@ public void TestEnvironmentProperty()
             });
         }
 
-        [ActiveIssue(14417)]
```

Comment:
This can't be re-enabled until after packages are updated with your fix, or alternatively you need to add a temporary P2P reference from System.Diagnostics.Process to System.Runtime.Extensions.